touchforms/formplayer/static/formplayer/style/webforms.css

```diff
@@ -93,11 +93,17 @@ input:invalid {
 }
 
 
-.required .caption:before {
+.required .caption::before {
```

Comment:
do these both work? i've only seen `::` but obviously this was working before with just `:`.